tools/istio-docker.mk

```diff
@@ -60,11 +60,7 @@ $(ISTIO_DOCKER)/certs:
 
 # tell make which files are copied from the source tree and generate rules to copy them to the proper location:
 # TODO(sdake)                      $(NODE_AGENT_TEST_FILES) $(GRAFANA_FILES)
-DOCKER_FILES_FROM_SOURCE:=tools/packaging/common/istio-iptables.sh tools/packaging/common/istio-clean-iptables.sh \
-                          tests/testdata/certs/cert.crt tests/testdata/certs/cert.key tests/testdata/certs/cacert.pem
-# generates rules like the following:
```

Comment:
Keep the comments. They explain the rules generated by the `foreach` below.

Reply:
I think we don't need. Its already covered here: https://github.com/istio/istio/pull/18962/files#diff-d4ccc43f318f9d501d556d3a89f8f54dL52